File: Section1 - Basics/Math/problem4.js
// Brute Force
function lcmAndGcd(a, b) {
	//code here
	let maxi = Math.max(a, b);
	let lcm = 0;
	let gcd = 0;
	while (true) {
		if (maxi % a == 0 && maxi % b == 0) {
			lcm = maxi;
			break;
		}
		maxi++;
	}
	gcd = (a * b) / lcm;
	console.log(lcm, gcd);
}
lcmAndGcd(14, 8);

File: Section1 - Basics/Math/problem4.java
// Brute Force
Long[] lcmAndGcd(Long A , Long B) {
    // code here
    long maxi = Math.max(A, B);
    long lcm = 0;
    long gcd = 0;
    while (true) {
      if (maxi % A == 0 && maxi % B == 0) {
        lcm = maxi;
        break;
      }
      maxi++;
    }
    gcd = (A * B) / lcm;
    Long[] ans = new Long[2];
    ans[0] = lcm;
    ans[1] = gcd;
    return ans;
}

/* Note
The return type of method is array of Long not long or int
*/